hal/tpm_io_infineon.c

```diff
@@ -43,6 +43,7 @@
        defined(WOLFTPM_SWTPM) ||     \
        defined(WOLFTPM_WINAPI) )
 
+#if defined(CY_USING_HAL)
```

Comment:
It needs to be `#if defined(WOLFTPM_INFINEON_TRICORE) || defined(CY_USING_HAL)`

Reply:
good catch. updated.